src/suites/cts/copyImageBitmapToTexture.spec.ts

```diff
@@ -0,0 +1,94 @@
+export const description = `
+copy imageBitmap To texture tests.
+`;
+
+import { TestGroup } from '../../framework/index.js';
+
+import { GPUTest } from './gpu_test.js';
+
+export const g = new TestGroup(GPUTest);
+
+function calculateRowPitchForImageBitmap(width: number, bytesPerPixel: number): number {
```

Comment:
nit: this function isn't really specific to ImageBitmaps, maybe rename to "calculateRowPitch"

Reply:
Done.